tests/functional/server/testListMetrics.js

```diff
@@ -18,43 +19,106 @@ describe('Request ranges', function test() {
 
     const tests = [
         {
-            start: 0,
-            end: ((MAX_RANGE_MS / 60) - (1000 * 60) * 15) - 1,
+            action: 'ListMetrics',
+            type: 'buckets',
+            resource: 'my-bucket',
+            timeRange: [
+                0,
+                ((MAX_RANGE_MS / 60) - (1000 * 60) * 15) - 1,
+            ],
+            expected: {
+                buckets: 'bucket',
+            },
         },
         {
-            start: 0,
-            end: MAX_RANGE_MS - 1,
+            action: 'ListMetrics',
+            type: 'buckets',
+            resource: 'my-bucket',
+            timeRange: [
+                0,
+                MAX_RANGE_MS - 1,
+            ],
+            expected: {
+                buckets: 'bucket',
+            },
         },
         {
-            start: 0,
-            end: (MAX_RANGE_MS + (1000 * 60) * 15) - 1,
+            action: 'ListMetrics',
+            type: 'buckets',
+            resource: 'my-bucket',
+            timeRange: [
+                0,
+                (MAX_RANGE_MS + (1000 * 60) * 15) - 1,
+            ],
+            expected: {
+                buckets: 'bucket',
+            },
         },
         {
-            start: 0,
-            end: (MAX_RANGE_MS * 12) - 1,
+            action: 'ListMetrics',
+            type: 'buckets',
+            resource: 'my-bucket',
+            timeRange: [
+                0,
+                (MAX_RANGE_MS * 12) - 1,
+            ],
+            expected: {
+                buckets: 'bucket',
+            },
+        },
+        {
+            action: 'ListRecentMetrics',
+            type: 'buckets',
+            resource: 'my-bucket',
+            expected: {
+                buckets: 'bucket',
+            },
+        },
+        {
+            action: 'ListRecentMetrics',
+            type: 'accounts',
+            resource: '1234567890',
+            expected: {
+                accounts: CANONICAL_ID,
+            },
         },
     ];
 
     tests.forEach(test => {
-        const { start, end } = test;
-        it(`should handle a request range of ${end - start}ms`, done => {
-            const params = {
-                timeRange: [start, end],
+        const {
+            timeRange, type, resource, expected,
+        } = test;
+        const msg = timeRange
+            ? `should handle a request range of ${timeRange[0] - timeRange[1]}ms`
```

Comment:
I think it should be `${timeRange[1] - timeRange[0]}ms`
```suggestion
            ? `should handle a request range of ${timeRange[1] - timeRange[01]}ms`
```